backend/src/main/java/com/scottlogic/GMSv2/config/Config.java

```diff
@@ -0,0 +1,23 @@
+package com.scottlogic.GMSv2.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+
+@EnableWebSecurity
+@Configuration
+public class Config {
+
+  @Bean
+  public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
+    http
+        .authorizeRequests()
+        .antMatchers("/*").permitAll()
```

Comment:
Be careful that this doesn't overwrite your CORS changes! I think git should pick up the conflict though...